frontend/www/js/omegaup/common/footer_v2.ts

```diff
@@ -0,0 +1,36 @@
+import Footer from '../components/common/Footerv2.vue';
+import { OmegaUp } from '../omegaup';
+import { types } from '../api_types';
+import Vue from 'vue';
+
+OmegaUp.on('ready', () => {
+  const payload = types.payloadParsers.CommonPayload('payload');
+  let isLoggedIn = false;
+  let omegaUpLockDown = false;
+  if (
+    typeof payload !== 'undefined' &&
+    typeof payload.isLoggedIn !== 'undefined'
+  ) {
+    isLoggedIn = payload.isLoggedIn;
+    omegaUpLockDown = payload.omegaUpLockDown;
+  }
+
+  const commonFooter = new Vue({
+    el: '#common-footer',
+    render: function(createElement) {
+      return createElement('omegaup-common-footer', {
+        props: {
+          isLoggedIn: this.isLoggedIn,
+          omegaUpLockDown: this.omegaUpLockDown,
```

Comment:
```suggestion
          isLoggedIn: (payload && payload.isLoggedIn) || false,
          omegaUpLockDown: (payload && payload.omegaUpLockDown) || false,
```

como no se está modificando dinámicamente ninguno de estos dos valores, no es necesario hacer el one-way binding con `this.`. Y se le puede dar inline a esos valores de paso!